arch/arm/Kconfig

```
@@ -1142,6 +1142,19 @@ config TARGET_HIKEY
 	  Support for HiKey 96boards platform. It features a HI6220
 	  SoC, with 8xA53 CPU, mali450 gpu, and 1GB RAM.
 
+config TARGET_AX3000_EVK
+	bool "Support Axiado AX3000 Chip"
+	select ARM64
+	select DM
+	select DM_SERIAL
+	select ZYNQ_SERIAL
+	select GPIO_EXTRA_HEADER
+	select MMC_SDHCI_AXIADO
+	select PHY_AXIADO_EMMC
+	help
+	  Support for Axiado AX3000 version1.0. AX3000 is a quad-core
```

```suggestion
	  Support for Axiado AX3000. AX3000 is a quad-core
```
I'm not sure if we should disclose the `Version 1.0`

---

Done.